ch04/README.md

```diff
@@ -362,7 +362,7 @@ int i; double d; const string *ps; char *pc; void *pv;
 pv = (void*)ps; // pv = const_cast<string*>(ps); or pv = static_cast<void*>(const_cast<string*>(ps));
 i = int(*pc);   // i = static_cast<int>(*pc);
 pv = &d;        // pv = static_cast<void*>(&d);
-pc = (char*)pv; // pc = reinterpret_cast<char*>(pv); 
+pc = (char*)pv; // pc = static_cast<char*>(pv);   ref: Page 163.
 ```
```

Comment:
Yes, as we take `(c) pv = &d ` into consideration, `pv`s original pointer type is `double`.
Ok I'll revert this fix .Thanks.